cc.config.json

```diff
@@ -678,6 +678,12 @@
             "type": "boolean",
             "value": false,
             "internal": true
+        },
+        "CULL_MESHOPT": {
```

Comment:
Where to set the value?

Reply:
in editor 
https://github.com/cocos/cocos-editor/pull/2518

Reply:
In what situation needs to cull meshopt? And what situation doesn't need to cull?

Reply:
cull meshopt in order to reduce the engine package size.
but with meshopt, we can reduce mesh assets size.